src/app/pages/partners/active-filters/filter-remover.js

```diff
@@ -4,7 +4,7 @@ export default function FilterRemover({label, store, value}) {
     return (
         <div className="filter-remover">
             <span>{label}</span>
-            <button type="button" onClick={() => store.toggle(value)}>
+            <button type="button" aria-label="remove" onClick={() => store.toggle(value)}>
```

Comment:
I've just discovered there's a new `data-action` attribute for things like this, you could add `data-action="delete"`: https://www.w3.org/TR/adapt-content/#action-explanation.